CHANGES.md

```diff
@@ -1,3 +1,43 @@
+## edge-19-2.4
+
+* CLI
+  * Implemented `proxy-cpu-limit` and `proxy-memory-limit` for setting the
+    proxy resources limits (`proxy-cpu` and `proxy-memory` were deprecated in
+    favor of `proxy-cpu-request` and `proxy-memory-request`) (thanks @TwinProduction!)
+  * Updated the `inject` and `uninject` subcommands to issue warnings when
+    resources lack a `Kind` property (thanks @Pothulapati!)
+  * Unhid the `install-cni` command and its flags, and tweaked their descriptions
+  * Fixed handling of kubeconfig server urls that include paths
+  * Added the `doc` command for generating docs for the CLI commands
+  * Removed the "experimental" flag from auto-inject
+  * Updated the `profile` help text to match the other commands
+  * Added the flags `-update` and `-pretty-diff` to tests to allow overwriting
+    fixtures and to print the full text of the fixtures upon mismatches
+  * Added the "ep" alias for the `endpoints` command (also @Pothulapati!)
+  * Split the install templates by component
+  * Added a validator for the `proxy-log-level` flag
+  * Renamed a function in a test that was shadowing a go built-in function
+    (thanks @huynq0911!)
+  * Fixed sporadic (and harmless) race condition error in `linkerd check`
+  * Introduced golangci-lint tooling, using `.golangci.yml` to centralize
+    the config
+  * Added a `-cover` parameter to track code coverage in go tests
+    (more info in TEST.md)
+  * Added integration tests for `single-namespace`
```

Comment:
Let's also move these last three bullet points to the "Internal" section at the bottom of the changelog, since none of them are related to the CLI.